tests/functional_small/solvers_nonlinear/updater/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+
+if (PRESSIO_ENABLE_TPL_EIGEN)
+  set(SRC1 ${CMAKE_CURRENT_SOURCE_DIR}/basic_backtrack.cc)
+  add_serial_utest(${TESTING_LEVEL}_basic_backtrack ${SRC1})
```

Comment:
this is missing the ${ROOTNAME} in it

Reply:
Can you add this? Not sure what it should look like.

Reply:
@eparish1 
should read: `add_serial_utest(${TESTING_LEVEL}_basic_backtrack ${ROOTNAME} ${SRC1})`

Reply:
Any idea I get this error when I do that?

CMake Error at cmake/macrosForCreatingUnitTests.cmake:7 (add_executable):
  Cannot find source file:

    functional_small_solvers_nonlinear

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc
Call Stack (most recent call first):
  tests/functional_small/solvers_nonlinear/updater/CMakeLists.txt:4 (add_serial_utest)


CMake Error at cmake/macrosForCreatingUnitTests.cmake:7 (add_executable):
  No SOURCES given to target:
  functional_small_backtrack_strictly_decreasing_objective
Call Stack (most recent call first):
  tests/functional_small/solvers_nonlinear/updater/CMakeLists.txt:4 (add_serial_utest)